golden_bridge.rb

```diff
@@ -0,0 +1,61 @@
+class Array
+  def convert_to_integer
```

Comment:
Would you use similar code in production code?
By `similar code`, I mean defining methods on the `Array` class.
If not, what is the alternative?